packages/webapp/src/containers/Animals/SingleAnimalView/index.tsx

```diff
@@ -149,45 +164,79 @@ function SingleAnimalView({ isCompactSideMenu, history, match, location }: AddAn
       console.error(e);
       dispatch(enqueueErrorSnackbar(t('message:ANIMALS.FAILED_UPDATE_BATCH')));
     }
+  };
+
+  const getInventoryId = () => {
+    if (!selectedAnimal && !selectedBatch) {
```

Comment:
Since there's a redirect to another page on the effect above if the batch or animal doesn't exist, is this check necessary?

Reply:
I think I was hoping to avoid doing `(selectedAnimal || selectedBatch)!` on line 175. It didn't work as expected, so I removed it!